src/goto-symex/symex_other.cpp

```diff
@@ -187,7 +187,9 @@ void goto_symext::symex_other(
     // process_array_expr)
     // 3. use the type of the resulting array to construct an array_of
     // expression
-    DATA_INVARIANT(code.operands().size() == 2, "array_set takes two operands");
+    DATA_INVARIANT(
```

Comment:
Ditto here - this change doesn't make sense to me because the new message is more generic, and the indentation is wrong.